apps/web/src/views/CakeStaking/components/LockCakeForm.tsx

```diff
@@ -32,7 +32,7 @@ const CakeInput: React.FC<{
   const onInput = useCallback(
     (input: string) => {
       setPercent(null)
-      onUserInput(input)
+      onUserInput(input === '.' || input === '。' ? '0.' : input)
```

Comment:
hmm, seems no need to add ‘。’, 

as the input has `^[0-9]*[.,]?[0-9]{0,18}$` reg

Reply:
```suggestion
      onUserInput(input === '.' ? '0.' : input)
```